uholny-phaseout-eu

vrq-github opened the pull request.

vrq-github committed:
uholny-phaseout-eu

mukrop commented:
Aky je tu stav? Moj odhad je, ze to este nepreslo korekturami (?) a vidim, ze tam este nie je nahrana infografika (preto aj padaju testy).
Tym padom sa rusim ako _asignee_, oznacte ma na review, ked to bude ready :-).

vrq-github commented:
Vo štvrtok máme s Kristy call a zbehneme tam všetko co pozostávalo. Phase
out incl.

Dňa so 29. 5. 2021, 21:27 Martin Ukrop ***@***.***>
napísal(a):

> Aky je tu stav? Moj odhad je, ze to este nepreslo korekturami (?) a vidim,
> ze tam este nie je nahrana infografika (preto aj padaju testy).
> Tym padom sa rusim ako *asignee*, oznacte ma na review, ked to bude ready
> :-).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/faktaoklimatu/web-sk/pull/42#issuecomment-850887103>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASNGY2IWQFE7SONBY6KYONLTQE5ZXANCNFSM45LW2T2Q>
> .
>

mukrop commented:
Super, dik za info.

vrq-github commented:
Tuto musím počkať na Kristi, napísala som jej, tak sa snáď už stretneme časovo

LenkaSedlackova committed:
language brush + treba update asi

@vrq-github 
"V Českej republike a Slovinsku momentálne prebiehajú jednania o termíne a stratégii dosiahnutia uhoľného phase-outu."
stále aktuálne? čo u nás na SVK?

cena EU ETS povolenky "v minulom roku sa pohybovala okolo 25 €"
čo je *minulý* rok?

LenkaSedlackova commented:
> 
> 
> Tuto musím počkať na Kristi, napísala som jej, tak sa snáď už stretneme časovo

@vrq-github prosím ťa, čo s týmto? vidím na GDrive infografiku z jari, mali ste nakoniec ten call s Kristi?

mukrop committed:
Zmazaný súbor naviac

mukrop committed:
Pridaná infografika

mukrop reviewed (changes requested):
Temer hotovo, ale par veci este zostava:
* [x] V texte je formulacia k cenam povolenie, ktora je kvoli plynutiu casu uz nespravna. Vid https://github.com/faktaoklimatu/web-cz/issues/976
* [ ] Neviem odhadut, ako velmi sa situacia zmenila vdaka COP26. Ak sa to menilo, nemali by sme tam doplnit nejaky disclaimer ako sa diskutuje v https://github.com/faktaoklimatu/web-cz/issues/952?

Vedel by sa mi na oboje pozriet napriklad Tom @protivinsky?

LenkaSedlackova commented:
>     * [ ]  V texte je formulacia k cenam povolenie, ktora je kvoli plynutiu casu uz nespravna. Vid [faktaoklimatu/web-cz#976](https://github.com/faktaoklimatu/web-cz/issues/976)

ja som sa toto v SVK verzii už snažila fixnúť, pozerala som ceny kvót kdesi

protivinsky commented:
Co se tyce nevyresenych otazek:
- Cena povolenky: ta informace, ktera je v textu, dnes jiz rozhodne nedava smysl, cili to by urcite chtelo aktualizovat. Nevim, kdo vlastne pripravoval tu ceskou verzi explaineru a premyslim, jaky postup dava nejvetsi smysl - asi proste prepsat ten odstavec v CZ verzi a pote prelozit do SK? Navic si myslim, ze pri soucasnych cenach povolenek nemuze byt uhli dlouhodobe profitabilni, nehlede na jakakoli politicka jednani. Takze podle me uhelny phase-out v zasade do cca 5-10 let probehne z ekonomickych duvodu vsude, kde energetika spada pod EU ETS.
- Zaverecna komunikace COP26 rika, ze potrebujeme phase-down of coal and phase-out of inefficient fossil fuel subsidies. Ale nic konkretnejsiho pro evropske zeme tam podle me nebylo.

mukrop commented:
Ako sme si pisali na Slacku, priprav prosim opravu pre cesky web a potom to prelozime do slovenskeho.

Inak zacali sme pouzivat varovny infobox pre casovo viazane infornacie, vid https://faktaoklimatu.cz/explainery/pozice-akteru-cop Ak uznas za vhodne, mozes take niecio zakomponovat do upravy.

Dlhodobo totiz nemozme mat vsetko 100% aktualne, to nejde. Ak presne cisla nie su nutne, smeroval by som k tomu pisat texty "nadcasovo" a konkretnym (napriklad cenam povoleniek) sa skor vyhybal. Ak su konkretne cisla uzitocne/nutne, pisal by som to hned viazane na datum (akoze "situacia k datumu XX bola taka a taka" napisat uz pocas vydania).

mukrop commented:
@LenkaSedlackova, novy odstavec navrhnuty v https://github.com/faktaoklimatu/web-cz/pull/982, budem rad za preklad.

LenkaSedlackova committed:
Tomov update

LenkaSedlackova commented:
@protivinsky "neboť" je pre mňa asi to, čo je pre teba "emisná kvóta" :D
každopádne hotofson! myslím, že je to óká
